### PR TITLE
fix: Dynamic modal copy based on context

### LINKED ISSUE #8134: Dynamic modal copy based on context
- currently, the copy for auto teleport does not change based on action 
- it shows teleport and confirm purchase

image
image

actions: 

## Listing 
button: Teleport and confirm listing(s)
copy inside signing: Bridging 10 DOT to list NFT/s on “Destination”

## minting item 
button: Teleport and mint NFT
copy inside signing: Bridging 10 DOT to mint NFT on “Destination”

## Minting collection 
button: Teleport and create collection
copy inside signing: Bridging 10 DOT to create collection on “Destination”

## buying 
- stays the same

## Transfer 
button: Teleport and Transfer NFT
copy inside signing: Bridging 10 DOT to transfer NFT on “Destination chain”

destination is the chain the action is happening on

### PR DESCRIPTION
**Pull request by Jarsen136**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

  👇 __ Let's make a quick check before the contribution.

  ## PR Type

  - [x] Bugfix

  ## Needs QA check

  - @kodadot/qa-guild please review

  ## Context

  - [x] Closes #8134
  - [ ] Requires deployment <snek/rubick/worker>

  #### Did your issue had any of the "$" label on it?

  - [x] My DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z)

  ## Screenshot 📸

  - [ ] My fix has changed UI

  ## Copilot Summary
  
### <samp>🤖[[deprecated]](https://githubnext.com/copilot-for-prs-sunset) Generated by Copilot at fbb483f</samp>

This pull request enhances the user interface and localization of the auto-teleport feature for NFT transactions. It adds different confirmation messages for the action button, updates the localization file with new and improved strings, and adds fallback values for the modal dialog.

  
### <samp>🤖[[deprecated]](https://githubnext.com/copilot-for-prs-sunset) Generated by Copilot at fbb483f</samp>

> _Sing, O Muse, of the skillful coder who devised_
> _A wondrous feature for the auto-teleport action,_
> _That lets the users swiftly move their NFTs across_
> _The blockchain realms with ease and satisfaction._

### CODE REVIEW

File: components/common/autoTeleport/AutoTeleportActionButton.vue [hunk 1]
```diff
@@ -191,6 +191,15 @@ const hasNoFundsAtAll = computed(
   () => !hasEnoughInCurrentChain.value && !hasEnoughInRichestChain.value,
 )
 
+const confirmButtonTitle = computed(() => {
+  const interaction =
+    transactions.value.actions[0].interaction?.toLocaleLowerCase()
+  return (
+    $i18n.t(`autoTeleport.steps.${interaction}.confirm`) ||
```
Comment (preschian) [inline components/common/autoTeleport/AutoTeleportActionButton.vue:198]: I just want to confirm. Is it safe to return this? Afaik `$i18n.t` always returns `true(string)`, so it won't return `autoTeleport.steps.common.confirm.`

Comment (Jarsen136) [reply components/common/autoTeleport/AutoTeleportActionButton.vue]: Good point. It does return the original key (string). It looks like we do not have a simple way to set the fallback value for translation. I have removed the `autoTeleport.steps.common.confirm.`

**Comment by prury** [comment]
listing - ok 
minting item - ok
minting collection - ok
buying - ok
transfer nft - i don't think we have teleport enabled for it

**Comment by yangwao** [comment]
Thanks!
pay 30 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $30 @ 5.13 USD/DOT ~ 5.848 $DOT
🧗 [16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z](https://polkadot.subscan.io/account/16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z)
🔗 [0x49ebbcab1275264ca396c4e5731937279771816baaab0d0b61393032480da9ea](https://polkadot.subscan.io/extrinsic/0x49ebbcab1275264ca396c4e5731937279771816baaab0d0b61393032480da9ea)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**